lib/kwtool/crawler/upload_parser.ex

```diff
@@ -15,5 +15,11 @@ defmodule Kwtool.Crawler.UploadParser do
     end
   end
 
-  def parse(_), do: {:error, :file_is_invalid}
+  def parse(_), do: raise("Uploaded keyword file is invalid")
```

Comment:
Maybe we should change to `parse!` as tthis method can raise an exception now. what do you think?